src/wmtk/TriMeshOperationExecutor.hpp

```diff
@@ -7,7 +7,7 @@ namespace wmtk {
 class TriMesh::TriMeshOperationExecutor
 {
 public:
-    TriMeshOperationExecutor(TriMesh& m, const Tuple& operating_tuple);
+    TriMeshOperationExecutor(TriMesh& m, const Tuple& operating_tuple, Accessor<long>& hash_acc);
```

Comment:
keep the other constructor just for now (you can annotate with deprecated if you want)

Reply:
as mentioned in the previous comment, it is not possible because the accessor is a reference